test/test_regression_cli.py

```diff
@@ -46,18 +49,17 @@ def get_csv_md5(dataset, engines, tmpdir):
 
 def setup_module():
     """Update retriever scripts and cd to test directory to find data"""
-    os.chdir(os.path.dirname(os.path.abspath(__file__)))
-    os.system("retriever update")
+    os.chdir(site_dir)
+    os.system('cp -r {0} {1}'.format(os.path.join(site_dir, "test/raw_data"), site_dir))
```

Comment:
This step is important unless we move the raw_data directory from the test to the  retriever main directory. I prefer leaving it in test and just copy it